src/test/java/com/openshift/client/portforwarding/PortForwardingL.java

```diff
@@ -64,6 +64,15 @@ public static void main(String[] arg){
   }
 
   public static class MyUserInfo implements UserInfo, UIKeyboardInteractive{
+    String passwd;
+    JTextField passwordField=(JTextField)new JPasswordField(20);
+    final GridBagConstraints gbc = 
+            new GridBagConstraints(0,0,1,1,1,1,
+                                   GridBagConstraints.NORTHWEST,
+                                   GridBagConstraints.NONE,
+                                   new Insets(0,0,0,0),0,0);
+    private Container panel;
+    
```

Comment:
-1, this is apparently code imported from jsch? If so I would not touch a single bit of it.